### PR TITLE
Adding keybindings for sticky scroll

### LINKED ISSUE #169107: Add a command to reveal line shown in sticky scroll
Currently, it is only possible to interact with the sticky scroll lines via the mouse. We should also have a command that allows to do so, which can be bound to a keybinding to allow for keyboard only interaction.

### PR DESCRIPTION
**Pull request by aiday-mar**
Adding keybindings for sticky scroll. The keybindings chosen are: control + alt + shift + S. Fixes #169107.

### CODE REVIEW

File: src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts [hunk 1]
```diff
@@ -30,7 +32,15 @@ export class ToggleStickyScroll extends Action2 {
...
 				{ id: MenuId.StickyScrollContext }
-			]
+			],
+			precondition: undefined,
+			keybinding: {
+				weight: KeybindingWeight.EditorContrib,
+				primary: KeyMod.Shift | KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyS,
+				mac: {
+					primary: KeyMod.Shift | KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyS,
+				},
```
Comment (Tyriar) [inline src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts:42]: This isn't needed as primary applies

File: src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts [hunk 2]
```diff
@@ -30,7 +32,15 @@ export class ToggleStickyScroll extends Action2 {
 				{ id: MenuId.CommandPalette },
 				{ id: MenuId.MenubarViewMenu, group: '5_editor', order: 2 },
 				{ id: MenuId.StickyScrollContext }
-			]
+			],
+			precondition: undefined,
```
Comment (Tyriar) [inline src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts:36]: Think this is redundant

**Review by Tyriar — CHANGES_REQUESTED** [review]

**Comment by aiday-mar** [comment]
Thanks @Tyriar, I'll make this a draft pull request and bring it up in the UX sync this week :)

File: src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts (hunk 1 above)
Comment (aiday-mar) [reply src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts:42]: Thanks for the tip

**Pull request closed without merging**